fixtures/login-res.xml

```diff
@@ -1,10 +1 @@
-<?xml version='1.0'?>
-<methodResponse>
-  <params>
-    <param>
-      <value>
-        <string>sa/1EWr40BWU+Pq3VEOOpD4cQtxkeMuFUw==</string>
-      </value>
-    </param>
-  </params>
-</methodResponse>
```

Comment:
What's the reason for this change? Before it was more readable. Now it's a really long line and makes it more difficult to follow.